.github/workflows/cd-docs.yaml

```diff
@@ -1,7 +1,8 @@
-name: Deploy Documentation on new release
+name: Deploy documentation on new tag
 on:
-  release:
-    types: [created]
+  create:
+    tags:
+      - v*
```

Comment:
I think this might need to be `- 'v*'` (based on https://github.community/t/how-to-run-github-actions-workflow-only-for-new-tags/16075/10).

Reply:
Updated.